src/uk/gov/hmcts/contino/GradleBuilder.groovy

```diff
@@ -76,9 +76,9 @@ class GradleBuilder extends AbstractBuilder {
   }
 
   def securityCheck() {
-    steps.withCredentials([steps.usernamePassword(credentialsId: 'owasp-db-login', passwordVariable: 'OWASPDB_ACCOUNT', usernameVariable: 'OWASPDB_PASSWORD')]) {
+    steps.withCredentials([steps.usernamePassword(credentialsId: 'owasp-db-login', passwordVariable: 'OWASPDB_PASSWORD', usernameVariable: 'OWASPDB_ACCOUNT')]) {
```

Comment:
maybe swap username and password around so its less confusing?
normally you go username:password

I assume that's now this accident happened in the first place...